pkg/windows/windows.go

```diff
@@ -507,6 +509,35 @@ func (vm *windows) ConfigureWICD(watchNamespace, wicdKubeconfigContents string)
 	return nil
 }
 
+func (vm *windows) RestoreAWSRoutes() error {
+	ec2LaunchV2ServiceName := "\"Amazon EC2Launch\""
+	serviceExists, err := vm.serviceExists(ec2LaunchV2ServiceName)
+	if err != nil {
+		return fmt.Errorf("error checking if %s service exists: %w", ec2LaunchV2ServiceName, err)
+	}
+	// We don't want ensureServiceIsRunning to create the service if it does not exist, so we return without an error.
+	// Returning an error does not make sense as we could have an AMI configured without the EC2Launch service present
+	// and the route created using some other customer specific method which is unknown to us.
+	if !serviceExists {
+		vm.log.Info("missing", "service", ec2LaunchV2ServiceName)
```

Comment:
Wondering if this should be logged with lower versatility (i.e. WARN) now that states something is missing. Before with `service not present` it was fine. IMO.

Reply:
There is no `log.Warn` option. Are you asking for it to be `log.V(1).Info`?

PS: please don't give lgtm and then suggest a change.

Reply:
I also want to call out that we always want to see this message and it should not have an option to be suppressed.

Reply:
Understood.